### DISCUSSION TITLE
I am having trouble getting the plugin to work in Godot 4.0.

### DISCUSSION THREAD

**Discussion by lubkz** [dialogic-godot/dialogic#2365] [Q&A]
I am having trouble getting the plugin to work in Godot 4.0. I have tried multiple times, downloaded both newer and older versions, but I can't seem to get it to work. It always displays a message saying 'Can't add Autoload: Invalid name. Must not collide with an existing global script class name.' when i enable the Dialogic and I end up not knowing what to do, regardless of the version. Please help!

And, yes, i have followed the correct procedures as far as I know. I separated the "addons" folder and placed it in the same location as the main folders. The plugin appears in the list and I can mark it as enabled, but I am unable to do so due to this issue.

**Comment by Jowan-Spooner** [comment]
Hey @lubkz,
it's hard to tell what the problem might be. If you are indeed trying to use it with godot 4.0, then you probably have to use 4.2+ for the newer versions of dialogic 2 (alpha 13 and beyond). 
However 4.2 might actually be what you meant. In that case this is usually caused by clash with the class_name cache. You can try to fix it by making sure to have no old version of dialogic in your project, then deleting the .godot folder or just the `global_script_class_cache.cfg` in that folder, best be done while the project is closed then reopening the project. 

Hope this helps, good luck!
